locales-pending/premium.ftl

```diff
@@ -291,6 +291,14 @@ premium-upsell-dialog-title = Choose the level of protection that’s right for
 guided-resolution-flow-exit = Return to dashboard
 guided-resolution-flow-back-arrow = Go to previous step
 guided-resolution-flow-next-arrow = Go to next step
+# Variables
+# $estimated_time is the amount of time it would take for a user to manually resolve a leaked password breach. It will always be a number greater than 1.
+# "Est." is shortform for "Estimated".
+# "mins" is shortform for "minutes".
```

Comment:
If we're abbreviating this, does that mean this sentence needs to be short? If so, might be worth a callout for potential future localisers.